DataFormats/CTPPSReco/src/classes_def.xml

```diff
@@ -90,6 +90,20 @@
   <class name="edm::DetSetVector<CTPPSPixelRecHit>"/>
   <class name="edm::Wrapper<edm::DetSetVector<CTPPSPixelRecHit> >"/>
 
+<class name="CTPPSPixelLocalTrack" ClassVersion="4">
+    <version ClassVersion="3" checksum="2597505479"/>
+    <version ClassVersion="4" checksum="1279766578"/>
```

Comment:
please drop the intermediate version with checksum 2597505479 and rename 4 -> 3

Reply:
done